wallets/provider-math-wallet/src/index.ts

```diff
@@ -45,6 +51,11 @@ export const connect: Connect = async ({ instance, meta }) => {
   const nonEvmResults = await getNonEvmAccounts(instance);
   results = [...results, ...nonEvmResults];
 
+  if (cosmosInstance) {
```

Comment:
Kindly insert this code into the function called getNonEvmAccounts.

Reply:
Done